service/routes.py

```diff
@@ -115,6 +115,26 @@ def get_shopcarts(shopcart_id):
     app.logger.info("Returning shopcart: %s", shopcart.id)
     return make_response(jsonify(shopcart.serialize()), status.HTTP_200_OK)
 
+
+######################################################################
+# DELETE A SHOPCART
+######################################################################
+@app.route("/shopcarts/<int:shopcart_id>", methods=["DELETE"])
+def delete_shopcart(shopcart_id):
+    """Deletes a shopcart
+        Args:
+            user_id (str): the user_id of the shopcart to delete
+        Returns:
+            str: always returns an empty string
+    """
+    app.logger.info("Start deleting shopcart %s...", shopcart_id)
+    shopcart = Shopcart.get_by_id(shopcart_id)
+    if shopcart:
+        shopcart.delete()
+        app.logger.info("Shopcart deleted with id= %s ", shopcart_id)
+    return make_response("", status.HTTP_204_NO_CONTENT)
```

Comment:
In addition, please help to update the [Delete Shopcart API doc](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/blob/master/README.md?plain=1#L284), thanks! 🙇‍♀️